packages/protoc-gen-connect-query/package.json

```diff
@@ -21,7 +21,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "test": "jest",
-    "build": "pnpm clean && tsc --project tsconfig.build.json"
+    "build": "pnpm clean && tsc --project tsconfig.build.json",
+    "license-header": "license-header --ignore 'dist/**' --ignore 'coverage/**'"
```

Comment:
```suggestion
    "license-header": "license-header"
```

Reply:
I instead moved the license-header call to root, that way we don't have a bunch of little scripts doing the exact same thing.